Replace 'else if' with 'elseif'

akramulhasan opened the pull request:
<!--
Hi there! Thanks for contributing to WordPress!

Pull Requests in this GitHub repository **must** be linked to a ticket in the WordPress Core Trac instance (https://core.trac.wordpress.org), and are only used for code review. **No pull requests will be merged on GitHub.**

See the WordPress Handbook page on using PRs for Code Review more information: https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/

If this is your first time contributing, you may also find reviewing these guides first to be helpful:
- FAQs for New Contributors: https://make.wordpress.org/core/handbook/tutorials/faq-for-new-contributors/
- Contributing with Code Guide: https://make.wordpress.org/core/handbook/contribute/
- WordPress Coding Standards: https://make.wordpress.org/core/handbook/best-practices/coding-standards/
- Inline Documentation Standards: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/
- Browser Support Policies: https://make.wordpress.org/core/handbook/best-practices/browser-support/
- Proper spelling and grammar related best practices: https://make.wordpress.org/core/handbook/best-practices/spelling/
-->

<!-- Insert a description of your changes here -->
The WordPress coding standards recommend using 'elseif' instead of 'else if' for conditionals to ensure consistency in the code syntax. This is because 'else if' is not compatible with the colon syntax for if|elseif blocks.
However, in the WordPress core file wp-includes/atomlib.php, on lines 251, 263, 265, and 305, 'else if' is used for conditionals. To ensure consistency with the WordPress coding standards and improve the readability and maintainability of the code, I propose replacing all instances of 'else if' with 'elseif' on these lines.
This change will ensure that the code follows the recommended syntax and is consistent with the rest of the WordPress core codebase.
I will create a pull request to the WordPress core repository with this change.

Trac ticket: https://core.trac.wordpress.org/ticket/58063

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

akramulhasan committed:
Replace 'else if' with 'elseif'

In the WordPress core file wp-includes/atomlib.php, on lines 251, 263, 265, and 305, 'else if' is used for conditionals. I propose replacing all instances of 'else if' with 'elseif' on these lines.

mukeshpanchal27 reviewed (commented):
Thanks @akramulhasan, Can you search similar instance in whole folder and correct it so we don't get similar ticket for other files?

akramulhasan committed:
replaces all 'else if' with 'elseif'

In the WordPress core files, on several files 'else if' is found for conditionals. I propose replacing all instances of 'else if' with 'elseif'.

akramulhasan commented:
> Thanks @akramulhasan, Can you search similar instance in whole folder and correct it so we don't get similar ticket for other files?

Hi, @mukeshpanchal27 
Done!

jrfnl commented:
These are files from external dependencies which should not be changed.
